src/use-cases/queries/RecupererUneGouvernance.ts

```diff
@@ -19,6 +23,12 @@ export class RecupererUneGouvernance implements QueryHandler<Query, UneGouvernan
             .toArray(),
         },
       }))
+    const membres = await this.#repository.getMembres(query.uidUtilisateurCourant)
+    const peutVoirNotePrivee = Membre.gestionnairePeutVoirNotePrivee(membres, query.codeDepartement)
+    return peutVoirNotePrivee ? readModel : {
+      ...readModel,
+      peutVoirNotePrivee: false,
+    }
```

Comment:
```suggestion
    return {
      ...readModel,
      peutVoirNotePrivee,
    }
```

Ça ferait le job non ?